viewPost.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+include('config/init.php');
+
+
+$result=getAllBlogPosts();
+$blogPosts= getBlogPost($_GET['categoryId']);
+
+
+echoHeader("".$blogPosts['title'], "".$blogPosts ['description']);
```

Comment:
You shouldn't need the `"".` here. You can just pass `$blogPosts['title']` directly.

What you're doing here is making a new empty string and concatenating the title. But the title is already a string, so you don't need to make the empty one to start.

Reply:
I tried passing it as `$blogPosts['title]` and it didn't work. Instead I made `$blogPost['title']` into the variable `$title` and passed that through the function. Is that okay to do?